build/local/README.md

```diff
@@ -1,3 +1,13 @@
+## Prerequisites 
+Prior to following the instructions here, make sure of the following steps to ensure a smooth process - 
+ - You should have `go` installed on your machine. You can find the installation instructions for your OS [here](https://go.dev/doc/install). 
+    - Verify the installation by running `go version`.
+    - Add the path to go's installation `bin` folder to your PATH variable. Instructions to do this are also mentioned on the go installation instructions page.  
```

Comment:
is this not already done as part of installing go?

Reply:
Yes, this was indeed a part of installation steps, the steps mention - 
```
export PATH=$PATH:/usr/local/go/bin
```
Just in my case this did not work since for some reason go was not installed in `/usr/local/`. But yes, I think we can remove it since most devs are familiar with how PATH works.